src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -310,6 +310,7 @@ Apply to thead to make headers not bold.
 
 .text-preserve-space {
     white-space: pre-wrap;
+    word-break: break-all;
```

Comment:
Alright, the reason for `word-wrap` property didn't support was that table had `auto` layout. Once the `fixed-layout` is applied to the parent table, the `word-wrap` works like a charm breaking smartly. 

![snip20160728_30](https://cloud.githubusercontent.com/assets/794011/17227907/fadcb4da-5510-11e6-9dcd-7cc653e1dcb1.png)

Also, [auto table layout ](http://www.w3schools.com/cssref/pr_tab_table-layout.asp)ensures table width is set by widest unbreakable content, which explains our bug. Do you see any problem changing the question tables from auto to fixed-layout ?
@YongJieYongJie